t/exit.t

```diff
@@ -10,7 +10,7 @@ log_level('warn');
 repeat_each(120);
 #repeat_each(2);
 
-plan tests => repeat_each() * (blocks() * 5);
+plan tests => repeat_each() * (blocks() * 5) - (repeat_each() * 2);
```

Comment:
@thibaultcha I believe these two forms are mathematically identical and are both irrelevant to the value of `repeat_each()`. Am I overlooking anything here?